packages/eb-deep-merge/Jenkinsfile

```diff
@@ -0,0 +1 @@
+npmPackagePipeline {}
```

Comment:
Is this file still necessary? I don't think so, can we get rid of it?

Reply:
Fix in the other PR.